cob_gazebo_worlds/launch/world.launch

```diff
@@ -16,10 +16,13 @@
 	<env name="GAZEBO_MODEL_PATH" value="$(arg pkg_gazebo_worlds)/Media/models:$(optenv GAZEBO_MODEL_PATH)"/>
 
 	<!-- upload default configuration parameters -->
-	<include file="$(arg pkg_env_config)/upload_navigation_goals.launch">
+	<include file="$(find cob_default_env_config)/upload_navigation_goals.launch">
+		<arg name="pkg_env_config" value="$(arg pkg_env_config)"/>
```

Comment:
In order to avoid copy of that file in the external package. I added this way. Is this wrong?